swap_meet/vendor.py

```diff
@@ -1,2 +1,94 @@
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    '''
+    Vendor class has an attribute named inventory which starts as an empty list by default and complete 
+    methods such as adding, removing, swapping, and figuring out best categories in your inventory and 
+    your friend's inventory.
+    '''
+    
+    def __init__(self, inventory = None):
+        '''
+        Attribute named inventory which starts as an empty list by default, but can be pass in a list.
+        '''
+        if not inventory:
+            inventory = []
+        self.inventory = inventory
```

Comment:
Nice handling of the default empty inventory and remembering that we shouldn't pass a mutable object as a default param.

This can also be re-written as a ternary, as:

`my_var = "some value" if some_condition else "other value"`

In this situation, that would look like

```python
self.inventory = [] if inventory is None else inventory
```